modules/dxp/apps/segments/segments-asah-connector/src/main/java/com/liferay/segments/asah/connector/internal/client/AsahFaroBackendClientImpl.java

```diff
@@ -128,15 +129,10 @@ public void deleteExperiment(long companyId, String experimentId) {
 		}
 
 		try {
-			AnalyticsConfiguration analyticsConfiguration =
-				_analyticsSettingsManager.getAnalyticsConfiguration(companyId);
+			String path = StringUtil.replace(
+				_PATH_EXPERIMENTS_EXPERIMENT, "{experimentId}", experimentId);
 
-			_jsonWebServiceClient.doDelete(
-				analyticsConfiguration.liferayAnalyticsFaroBackendURL(),
-				StringUtil.replace(
-					_PATH_EXPERIMENTS_EXPERIMENT, "{experimentId}",
-					experimentId),
-				new HashMap<>(), _getHeaders(companyId));
+			_getResponse(companyId, Http.Method.DELETE, null, path, null);
```

Comment:
It is not a good practice to send null to a private method, as we need to handle null values, what do you think about use  Collection.emptyMap?

Reply:
Perfect, I have changed it and now we are not passing null values.